Changed the standalone base header and footer

higs4281 opened the pull request:
This mimics the standalone scheme we used for Retirement. 
It picks up static versions of the on-demand header/footer used on cfgov.
It also converts (again) to Python 2.7, Django 1.8 and Elasticsearch
## Additions
- static versions of the on-demand header/footer templates
## Testing
- Fire up with runserver 
## Review

any of @mistergone  @niqjohnson @marteki 
## Screenshots
## Todos
- footer icons don't display on some pages; may be a local issue
## Checklist
- [x] CHANGELOG has been updated

higs4281 committed:
Changed the standalone base header and footer

This mimics the standalone scheme we used for
Retirement. It picks up static versions of the
on-demand header/footer used on cfgov.

higs4281 committed:
update changelog

marteki commented:
Note: I had to run `./local_setup.sh` and revert my standalone to use Django 1.6 before I could get this running.

Look at those beautiful headers! 😍  I think the broken footer icons are due to icon fonts not loading; they're also not loading on the repay student debt tool.

![screen shot 2016-05-03 at 12 20 05 pm](https://cloud.githubusercontent.com/assets/1183435/14990481/c24b29a0-112a-11e6-872d-5a666dd599a5.png)

But... I noticed some oddness in the guide pages that doesn't seem related. 😞  Check out these page title sizes.

![screen shot 2016-05-03 at 12 19 19 pm](https://cloud.githubusercontent.com/assets/1183435/14990444/98d189ac-112a-11e6-966f-b0fcda121d0e.png)

![screen shot 2016-05-03 at 12 19 31 pm](https://cloud.githubusercontent.com/assets/1183435/14990447/9d46c394-112a-11e6-8e2b-de5e70d52055.png)

Seems like they're being reset by `footer.css`, and `guides.css` isn't being triggered last as it should. Take another look at the order of operations for the CSS in the template?

higs4281 committed:
Upgrade to Django 1.8 and Elasticsearch

Integrating with V1 leads us back to Django 1.8 and Elasticsearch.
This restores settings and requirements for that combination, and fixes
our README for starting up the project in standalone mode.

higs4281 committed:
travisty

higs4281 committed:
import haystack for backend

higs4281 committed:
import haystack and elasticsearch for backend

higs4281 committed:
Merge branch 'standalone' of github.com:cfpb/college-costs into standalone

higs4281 committed:
testing key

higs4281 committed:
fix secret

higs4281 committed:
remove unneeded imports

higs4281 committed:
use haystack 2.4.1

higs4281 committed:
add elasticsearch to README

Also renames the standalone setup script to `standalone_setup.sh`

amymok commented:
@higs4281 The only problem I have is I get an error when running standalone.py (part of the `gulp build`, but probably not related to this PR):

```
gulp-notify: [Compile Error] Error in plugin 'gulp-less'
Message:
    Invalid mapping: {"generated":{"line":1,"column":4613},"source":"/Users/moka/workspace/college-costs/src/vendor/cf-core/src/cf-media-queries.less","original":{"line":1,"column":null},"name":null} in file undefined line no. undefined
Details:
    lineNumber: undefined
    fileName: undefined
```

@niqjohnson @anselmbradford mentioned that we can comment out two lines in gulp/tasks/styles.js:
- 13: `.pipe( $.sourcemaps.init() )`
- 23: `.pipe( $.sourcemaps.write( '.' ) )`

But something we need to figure out to fix them instead of commenting out.

Also I don't have right to merge this PR.  Please sync up with the latest code as well.

anselmbradford commented on paying_for_college/templates/standalone/base_update.html:
```diff
@@ -201,187 +130,36 @@
+<script type="text/javascript" src="http://www.consumerfinance.gov/static/agreements/js/chosen.jquery.js?ver=all"></script>
 {% endblock %}
-
+<script src="http://www.consumerfinance.gov/static/nemo/_/js/functions.js?ver="></script>
```
Does this need a value set for `?ver=`?

higs4281 replied:
the empty query appears to be a no-op suffix. we could lose it.

higs4281 committed:
Merge branch 'master' of github.com:cfpb/college-costs into standalone

higs4281 committed:
drop deprecated nemo assets

higs4281 committed:
Merge branch 'standalone' of github.com:cfpb/college-costs into standalone
